catalog/spatial/wfs/2.0.0/spatial-wfs-v2_0_0-source/src/main/java/org/codice/ddf/spatial/ogc/wfs/v2_0_0/catalog/source/WfsSource.java

```diff
@@ -801,6 +806,10 @@ protected GetFeatureType buildGetFeatureRequest(Query query) throws UnsupportedQ
                             .getLocalPart();
                 }
 
+                if(StringUtils.isNotBlank(srsName)) {
```

Comment:
We don't want to set the srsName attribute at all if the value is empty, null, or whitespace only which is why I used https://commons.apache.org/proper/commons-lang/javadocs/api-2.6/org/apache/commons/lang/StringUtils.html#isNotBlank(java.lang.String)